cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -8,7 +8,11 @@ metadata:
   labels:
     application: gateway-operator
     component: fabric-gateway-controller
+  {{ if .ConfigItems.fabric_gateway_controller_version }}
+    version: "{{ .ConfigItems.fabric_gateway_controller_version }}"
+  {{ else }}
     version: "{{ $version }}"
+  {{ end }}
```

Comment:
Lets add fabric_gateway_controller_version into config-defaults with current version and remove $version form the top of the file